ddp-workspace/projects/ddp-brain/src/assets/i18n/en.json

```diff
@@ -935,5 +941,34 @@
             "ActivityEdit": "Click the Edit button to temporarily unlock for editing. Your edits will be saved as you go. When you click submit or navigate away, the activity will be locked again automatically.",
             "EditButton": "Edit"
         }
+    },
+    "Toolkit": {
+        "Common": {
+            "Organization": "Count Me In"
+        },
+        "Header": {
+            "HeaderLogo": "Metastatic<br>Breast Cancer<br>Project",
```

Comment:
should MBC's logo be used here? Why not Brain's?

Reply:
not referenced . cleaned up.